docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md

```diff
@@ -96,20 +96,34 @@ You can provide a dictionary as the `options` parameter of `build_batch_request(
 
 ### 3. Create a Validator
 
-We will use a Validator to interactively create our Expectations.  To do this, a Validator needs two parameters. One parameter indicates the Batch containing data that is used to Validate the Expectations. The other provides a name for the combined list of Expectations we create.
-
-```python name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py get_validator_and_inspect_data"
-```
+When you use a Validator to interactively create your Expectations, the Validator needs two parameters. One parameter identifies the Batch that contains the data that is used to Validate the Expectations. The second parameter provides a name for the combined list of Expectations you create.
 
 :::info Working outside a Jupyter Notebook
 
-If you're using a Jupyter Notebook you'll automatically see the results of the code you run in a new cell when you run the code. If you're using a different interpreter, you might need to explicitly print these results to view them:
+If you're using a Jupyter Notebook you'll automatically see the results of the code you run in a new cell when you run the code. If you're using a different interpreter, you might need to explicitly print these results to view them. For example:
 
 ```python name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py inspect_data_no_jupyter"
 ```
 
 :::
 
+1. Optional. Run the following command if you haven't created an Expectation Suite:
+
+    ```python 
+    context.create_expectation_suite("my_expectation_suite")
+    ```
+
+2. Optional. Run the following command if you haven't added or updated your Expectation Suite:
+
+    ```python 
+    context.add_or_update_expectation_suite("my_expectation_suite")
+    ```
```

Comment:
These python code blocks need to be pulled from an example script that runs the code.

Reply:
Additionally, only one of those lines is necessary.  I would stick with the first one: `create_expectation_suite`.  The error being addressed is cases when the Expectation Suite does not already exist, and for the purposes of creating it `create_expectation_suite` would be the more intuitive choice.

Reply:
@Rachel-Reverie I have removed the additional optional command as you recommended.